keyboards/model_f_labs/f62/keymaps/via/rules.mk

```diff
@@ -0,0 +1,6 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+BOOTMAGIC_ENABLE = no
+BOOTMAGIC_LITE = no
```

Comment:
This doesn't actually do anything.
```suggestion
```

Reply:
Removed from rules.mk

Added "void bootmagic_lite(void) {}" to xwhatsit_capsense_matrix.c